Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: install run test lint
+
+BIN = docker run \
+			--interactive \
+			--rm \
+			-v "/code" \
+			--name quarto-python \
+			quarto-python
+
+# Initialization ===============================================================
+
+install:
+	docker build --tag=quarto-python .
+
+# Run ===============================================================
+
+run:
+	 $(BIN) #./src/quarto.py
```

Comment:
Uncomment #./src/quarty.py here and remove CMD in the Dockerfile.
This way, you can run your docker image without run the program.